_data/students.yml

```diff
@@ -11,6 +11,14 @@
 #   lat: Student's Latitude
 #   lng: Student's Longitude
 
+
+- name: Shivam Rai  
+  country: India
+  github: ShivamRai2003
+  image: (rai)shivam.jpg 
+  twitter: Shivam Rai
+  
+  
```

Comment:
Looks good just remove this one extra line

Reply:
which line

Reply:
The two blank lines. Remove one of them